faculty/clients/environment.py

```diff
@@ -233,7 +233,7 @@ class EnvironmentSchema(BaseSchema):
     id = fields.UUID(data_key="environmentId", required=True)
     project_id = fields.UUID(data_key="projectId", required=True)
     name = fields.String(required=True)
-    description = fields.String(required=True)
+    description = fields.String(required=True, allow_none=True)
```

Comment:
We usually treat 'None' as equivalent to missing - can we make this `missing=None` instead of `required=True, allow_none=True`?

Reply:
Agreed. That's much clearer. I'll change it in the EnvironmentCreateUpdateSchema too for consistency